packages/react-components/react-search/src/components/SearchBox/useSearchBoxStyles.styles.ts

```diff
@@ -15,21 +15,21 @@ export const searchBoxClassNames: SlotClassNames<SearchBoxSlots> = {
 const useRootStyles = makeStyles({
   small: {
     columnGap: 0,
-    width: '468px',
+    maxWidth: '468px',
 
     paddingLeft: tokens.spacingHorizontalSNudge,
     paddingRight: tokens.spacingHorizontalSNudge,
   },
   medium: {
     columnGap: 0,
-    width: '468px',
+    maxWidth: '468px',
```

Comment:
looking at the spec, I think we want to use minWidth, right? It says the default width is 468px which I think that's why you have width, but maxWidth just limits how long it can get. Adding the minWidth will set the minimum width it can be and taking the similar effect of width.

Reply:
it's intended to let the width shrink ^^'